### DISCUSSION TITLE
How does npm-packager understand whether npm package should be listed in npm packager ui or not?

### DISCUSSION THREAD

**Discussion by FreePhoenix888** [deep-foundation/Discussions#13] [Q&A] [answered]
@Konstantin Dyachenko#8793 , How does npm-packager understand whether npm package should be listed in npm packager ui or not?

I see that my integration packages are listed in npm packager ui
Before I have been thinking that it depends on deep.json file but now I guess it depends on `deep-package` npm package tag. Should I remove it to make these packages now showed in the npm packager ui?  
![image](https://github.com/deep-foundation/Discussions/assets/66206278/be93c5fc-f889-4f2d-b76e-a088912dffd7)

**Comment by konard** [comment, marked as answer]
@FreePhoenix888 we use only `deep-package` keyword right now. You should not use this keyword if the package does not contain `deep.json`. This keyword automatically is added on publish of any package from inside Deep UI.
